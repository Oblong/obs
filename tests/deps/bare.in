oblong-greenhouse-gs4.5x-build-deps
oblong-gs4.5-stagingcore
oblong-staging4.5
