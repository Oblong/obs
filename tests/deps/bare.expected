oblong-gs4.5-stagingcore
oblong-staging4.5
